cni/network/multitenancy.go

```diff
@@ -249,13 +255,14 @@ func (m *Multitenancy) getNetworkContainersInternal(
 		return nil, []net.IPNet{}, fmt.Errorf("%w", err)
 	}
 
-	log.Printf("Network config received from cns %+v", ncConfigs)
+	log.Logger.Info("Network config received from cns", zap.Any("nconfig", ncConfigs))
```

Comment:
does this print ncConfig structure with field names or just values?

Reply:
The whole object